src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -189,6 +192,24 @@ def load_arguments(self, _):
         c.argument('encryption_in_transit', arg_group='Encryption In Transit', arg_type=get_three_state_flag(),
                    help='Indicates whether enable encryption in transit.')
 
+        # Autoscale Configuration
+        c.argument('autoscale_type', arg_group='Autoscale Configuration', arg_type=get_enum_type(['Load', 'Schedule']),
+                   help='The autoscale type.')
+        c.argument('autoscale_min_count', options_list=['--autoscale-min-count', '--autoscale-min-workernode-count'],
+                   arg_group='Autoscale Configuration', help='The minimal workernode count for Load-based atuoscale.')
+        c.argument('autoscale_max_count', options_list=['--autoscale-max-count', '--autoscale-max-workernode-count'],
+                   arg_group='Autoscale Configuration', help='The max workernode count for Load-based atuoscale.')
+        c.argument('timezone', arg_group='Autoscale Configuration', validator=validate_timezone_name,
+                   completer=get_generic_completion_list(AUTOSCALE_TIMEZONES),
+                   help='The timezone for schedule autoscale type. Values from `az hdinsight autoscale list-timezones`')
+        c.argument('days', nargs='+', arg_group='Autoscale Configuration',
+                   completer=get_generic_completion_list(week_days),
+                   help='A space-delimited list of schedule day. Valid days are {}.'.format(', '.join(week_days)))
+        c.argument('time', arg_group='Autoscale Configuration', validator=validate_time,
+                   help='The 24-hour time in the form xx:xx in days.')
```

Comment:
>the form [](start = 45, length = 8)

the form of?

Reply:
fixed.

---
In reply to: [466340274](https://github.com/Azure/azure-cli/pull/14692#discussion_r466340274) [](ancestors = 466340274)